pkg/scheduler/framework/plugins/helper/spread_test.go

```diff
@@ -97,8 +97,9 @@ func TestGetPodServices(t *testing.T) {
 			get, err := GetPodServices(fakeInformerFactory.Core().V1().Services().Lister(), test.pod)
 			if err != nil {
 				t.Errorf("Error from GetPodServices: %v", err)
-			} else if !reflect.DeepEqual(get, test.expect) {
-				t.Errorf("Expect services %v, but got %v", test.expect, get)
+			}
```

Comment:
leave the `else`. If there was an error, `get` would just be empty and not match.

Reply:
done.